### PR TITLE
mrc-3235 Include pyrrole interventions in Strategise

### PR DESCRIPTION
**Pull request by EmmaLRussell**
Ensure that pyrrole interventions are included in strategise when `netUse` value is not 0. 

Also removes mac build from the github action.

### CODE REVIEW

**Comment by EmmaLRussell** [comment]
Any idea what's causing the test coverage failure?

**Comment by richfitz** [comment]
coverage is failing because that is using a mac builder, I can patch that up if you'd like

File: R/import.R [hunk 1]
```diff
@@ -1,6 +1,6 @@
 mintr_db_download <- function(path, quiet = FALSE) {
   version <- mintr_data_version()
-  root <- "https://mrcdata.dide.ic.ac.uk/mint"
+  root <- "https://github.com/mrc-ide/mintr/releases/download/data-20230208"
   url <- sprintf("%s/%s.tar", root, version)
```
Comment (richfitz) [inline R/import.R:4]:
```suggestion
  root <- "https://github.com/mrc-ide/mintr/releases/download/"
  url <- sprintf("%s/data-%s/%s.tar", root, version, version)
```

though I imagine we'll update the pattern at some point, this is closer to the intent I think

**Review by richfitz — CHANGES_REQUESTED** [review]
actual changes look fine, just a tiny fix to help later

**Comment by EmmaLRussell** [comment]
Once this gets merged I think we can merge the epic branch since this will work with the newest MINT changes.

**Comment by EmmaLRussell** [comment]
Ah actually no, we'll want to do this first: https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-4149/Fix-mint-for-mintr-epic-branch

**Review by richfitz — APPROVED** [review]

**Pull request merged**